libs/webgui/pilight.js

```diff
@@ -1297,6 +1298,9 @@ $(document).ready(function() {
 			} else {
 				startWebsockets();
 			}
+			if('settings' in data && 'stats-enable' in data['settings']) {
+				bStatsEnabled = data['settings']['stats-enable'] == 1
```

Comment:
I don't think the `== 1` is necessary.

Reply:
Hmm, it seems it's unnecessary. Just wanted to be sure about truthy and false-y values in js first